tasks/fh-build.js

```diff
@@ -171,7 +172,7 @@ module.exports = function(grunt) {
         options: {
           mode: 'tgz',
           archive: 'dist/<%= fhpkg.name %>-<%= fhbuildver %>-<%= fhos %>.'
-            + process.arch + '.tar.gz'
+          + process.arch + '.tar.gz'
```

Comment:
Should this one not remain indented (I'm not sure, but it looks a little strange to me, I don't know what the rule is)?